operator/config/manager/manager.yaml

```diff
@@ -44,7 +44,7 @@ spec:
             - --leader-elect
             - --health-probe-bind-address=:8081
             - --program-fs-adv-addr=pulumi-kubernetes-operator.$(POD_NAMESPACE).svc.cluster.local:80
-            - --zap-log-level=error
+            - --zap-log-level=info
```

Comment:
Slipped in this fix, is now consistent with Helm chart and is more useful at info level.